msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md

```diff
@@ -305,7 +305,7 @@ Your bot can mention tags in text messages and Adaptive Cards posted in channels
 >
 > * Tag mentions are available only in [public developer preview](../../../resources/dev-preview/developer-preview-intro.md).
 > * Tag mentions are supported in Teams desktop and web clients. However, it's not supported in Teams mobile client.
-> * Tag mentions are supported in Government Community Cloud (GCC) and GCC-High tenants, and not supported in  Department of Defense (DoD) tenant.
+> * Tag mentions are currently not supported in Government Community Cloud (GCC), GCC-High, and Department of Defense (DoD) tenants.
```

Comment:
Remove currently.
Redraft: Tag mentions aren't supported in Government Community Cloud (GCC), GCC-High, and Department of Defense (DoD) tenants.

Reply:
Updated the change.